public/locales/ja/common.json

```diff
@@ -63,22 +63,22 @@
       "90d": "90日"
     },
     "target-table-select": {
-      "01": "Main table",
-      "02": "My table"
+      "01": "メインテーブル",
+      "02": "私のテーブル"
     }
   },
   "signin": {
     "signin": "サインイン",
     "signout": "サインアウト",
     "choose-app": "サインインするアプリを選択",
     "login-with": "Login with",
-    "sign-with": "Sign with",
+    "sign-with": "Login with",
```

Comment:
Login - is just to get an address and show information 
Sign - is to sign the transaction with Xumm or hardware wallet etc. To use a device/wallet to sign it. (to confirm)

Reply:
This is just my mistake😢